app/assets/javascripts/premadeModal.js

```diff
@@ -0,0 +1,18 @@
+var onReadyStrategiesModal = function() {
+	$("#myBtn").click(function() {
```

Comment:
We should use a more descriptive name instead of `#myBtn`, it should be something like `#add_premade_button`